### DISCUSSION TITLE
2024/03/05/Multifunction/

### DISCUSSION THREAD

**Discussion by giscus** [AbyssSkb/abyssskb.github.io#2] [Announcements]
# 2024/03/05/Multifunction/

多元函数的基本概念 NNN 维空间 称有序数组 (x1,x2,⋯ ,xn)(x_1, x_2, \cdots, x_n)(x1​,x2​,⋯,xn​) 为一个 nnn 维点（nnn 维向量）。所有 nnn 维点组成的集合称之为 nnn 维空间，记为 Rn\mathbb{R}^nRn。 NNN 维空间的距离 设 A(x1,x2,⋯ ,xn)A(x_1, x_2, \cdots, x_n)A(x1​,

https://blog.myabyss.top:8443/2024/03/05/Multifunction/?highlight=%E5%A4%9A%E5%85%83%E5%87%BD%E6%95%B0

**Comment by AbyssSkb** [comment]
test
